NEWS.md

```diff
@@ -16,6 +16,7 @@ New features:
 - Both the `pch` and `lty` arguments now accept a "by" convenience keyword for
 automatically adjusting plot characters and line types by groups (#28,
 @grantmcdermott).
+- Point-range plots with `type="pr"` (#35 @vincentarelbundock)
```

Comment:
I'm not entirely sure about "pr" as a type shorthand. Ofc we could always alias them, but I think new types---beyond those offered by base plot----should probably be defined as full words, e.g. `type = "pointrange"`. 

One benefit of the full word aliasing would be consistency with other new methods, e.g. as wishlisted by #21 (type = "density") or possibly even #29 (type = "symbols"). Thoughts?

This is kind of an aside, but I can envisage that these ultimately gain convenience wrapper methods like `hist2 = plot2(..., type = "historgram")`, etc.

Reply:
I like this a lot.